docs/conf.py

```diff
@@ -3,28 +3,28 @@
 # CvDisinfo-Detect documentation build configuration file, created by
 # sphinx-quickstart.
 #
-# This file is execfile()d with the current directory set to its containing dir.
+# This file is execfile() with the current directory set to its containing dir.
 #
 # Note that not all possible configuration values are present in this
 # autogenerated file.
 #
 # All configuration values have a default; values that are commented out
 # serve to show the default.
 
-import os
-import sys
+# import os (original imported these, however flake8 said they were unused)
```

Comment:
If you're not using, always remove them. Don't comment them out unless you *know for a fact* you're going to use them *soon*.